spec/mailers/staff/survey_response_completed_mailer_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Staff::SurveyResponseCompletedMailer, type: :mailer do
+  describe 'notify' do
+    let(:user) { create(:user, :with_primary_skill, :with_feedback) }
+    let(:mail) { Staff::SurveyResponseCompletedMailer.notify(user.id) }
+    let(:recipients) { create_list(:user, 2) }
+
+    it 'renders the subject' do
+      expect(mail.subject).to eq(I18n.t('dashboard.survey_responses.notices.completed'))
```

Comment:
`its(:subject) { is_expected.to eq I18n.t('dashboard.survey_responses.notices.completed') }`